scripts/delete_bundle.py

```diff
@@ -27,19 +26,8 @@ def main(argv):
 
 
 def delete_bundle(bundle_uuid, bundle_version):
-    logging.info('Deleting bundle %s.%s', bundle_uuid, bundle_version)
-    base_url = config.indexer_endpoint()
-    deletion_endpoint = f'{base_url}/delete'
-    notification = {
-        'match': {
-            'bundle_uuid': bundle_uuid,
-            'bundle_version': bundle_version
-        }
-    }
-    response = requests.post(url=deletion_endpoint,
-                             json=notification)
-    response.raise_for_status()
-
+    azul_client = AzulClient()
```

Comment:
The AzulClient instance should be shared across `delete_bundle` invocations. I think this can be achieved most easily by inlining this function (since it is now pretty anemic) and instantiating AzulCient outside of the bundle loop.